.github/workflows/velociy-dev-s3-deploy.yml

```diff
@@ -29,6 +29,14 @@ jobs:
     - name: Set Common Variables
       run: |
        # bash ./eng/environments/common.sh
```

Comment:
Looks like that script file doesn't exist

Reply:
This will nolonger be needed so I'm removing the step